package.json

```diff
@@ -37,13 +37,17 @@
     "@biomejs/biome": "1.9.4",
     "@changesets/changelog-github": "^0.5.1",
     "@changesets/cli": "^2.29.4",
+    "@storybook/addon-docs": "^9.0.8",
+    "@storybook/addon-vitest": "9.0.8",
+    "@storybook/react-vite": "^9.0.8",
```

Comment:
Code looks overall good, but wondering if there is any reason we need to move these to root?

Reply:
`addon-docs/blocks` became unreachable in various mdx-files we have scattered around (in both react and other places) unless i installed it in root.  `storybook`, `addon-vitest` and `react-vite` need to be in both react and storybook so I just moved them to root to avoid duplication, but these can be moved back to storybook and react if you wnat that setup instead